.github/dependabot.yml

```diff
@@ -9,7 +9,3 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
```

Comment:
Why the github actions were removed from the dependabot checks?

Also, for docker, we would need to specify like this so it groups the baseimage updates in one PR

```
- package-ecosystem: "docker"
  directory: "/"
  schedule:
    interval: "daily"
  groups:
     dev-dependencies:
        patterns:
          - "tiiuae/fog-ros-sdk"
          - "tiiuae/fog-ros-baseimage-builder"
          - "tiiuae/fog-ros-baseimage"
 ```

Reply:
Fixed with e295e736bad44f42265b49e88015f4dd1888d8e4

Reply:
Put back the githut action dependabot definition with eecc410f22fe418408ab0b9a8fd9a525ab7de8ee